source/includes/_fiat_to_crypto_flow.md

```diff
@@ -218,21 +218,50 @@ So the 2 potential cases are:
   - You can either not send the `kyc_verified` parameter or send it with a `null` value.
 
 
-### Status callback
+## Notification of successful transaction
 
-In order to receive status callbacks for an operation, a `callback_url` should be provided in the request. **ON/RAMP** will call back the merchant with the following JSON payload:
+A URL provided by the merchant in a previous request (see `callback_url` in [Create fiat to crypto invoice](#create-fiat-to-crypto-invoice)) will be used to send a notification request once the transaction has been confirmed.  The merchant is expected to respond with 200 status code.
+
+<aside class="success"><b><code>POST callback_url</code></b></aside>
+
+> Example cURL
+
+```shell
+
+curl -X POST callback_url \
+-H 'Content-Type: application/json' \
+-d '{
+    "invoice_id": "10e35c16-3ff8-4238-b5b5-34b4da9b4115",
+    "timestamp_utc": "2020-01-01 12:45:00.000000+00",
+    "status": {
+        "completed": {}
+    }
+}'
+```
+
+> Request JSON Body
 
 ```json
 {
     "invoice_id": "10e35c16-3ff8-4238-b5b5-34b4da9b4115",
     "timestamp_utc": "2020-01-01 12:45:00.000000+00",
-    "status": { 
+    "status": {
         "completed": {}
     }
 }
 ```
 
-> IMPORTANT: This callback is going to be done with **HTTP POST** method and not with GET.
+### Request JSON Fields
+
+| Field         | Type   | Description                                             |
+| --------------| -------| --------------------------------------------------------|
+| invoice_id    | String | Internal **ON/RAMP**'s invoice identifier.       |
+| timestamp_utc | Date   | Timestamp when transaction has been completed.          |
+| status        | Status | Contains "completed" field when transaction is completed. |
+
+
+### Expected Response
 
-The `status` field can have the following values:
-    - `completed`: The payment has been succesfully completed.
+| Status |
```

Comment:
This is where we will add the tx_hash if available?

Reply:
If we implement 2 callbacks then yes.
The simplest solution is to re-use fiat callback and include it under crypto_info, like this:
```
{
  "reference": "ac400127-93a9-4b9c-9612-c23a3c078933",
  "invoice_id": "62b570e8-9723-4287-a5a8-e825c2ffced2",
  "crypto_info": {
    "tx_hash": "220c540c074ff2744d28f7a37a89a55c8a080590a68da6c2a9d3460ec01b3385"
  }
}
```